common-rendering/src/components/MetaData.tsx

```diff
@@ -0,0 +1,363 @@
+		if (tag.type === "Contributor") {
+			names.push({ name: tag.title, id: tag.id });
+		}
+	});
+};
+
+/*function Byline(byline: string, authors: Authors[]) {
+	if (byline.includes("now")) {
+		byline.replace("now", "(now)");
+	}
+	if (byline.includes("earlier")) {
+		byline.replace("earlier", "(earlier)");
+	}
+}*/
+
+const ContributorLink: React.FC<{
+	contributor: string;
+	contributorTagId: string;
+}> = ({ contributor, contributorTagId }) => (
+	<a
+		rel="author"
+		data-link-name="auto tag link"
+		href={`//www.theguardian.com/${contributorTagId}`}
+	>
+		{contributor}
+	</a>
+);
+
+// Shamelessly borrowed from Lucy! (This should be lifted out as it's very useful!)
+function getThemeColours(theme: Theme) {
+	switch (theme) {
+		case Pillar.Sport:
+			return {
+				background: sport[100],
+				color: sport[100],
+			};
+		case Pillar.Culture:
+			return {
+				background: culture[200],
+				color: culture[200],
+			};
+		case Pillar.Lifestyle:
+			return {
+				background: lifestyle[200],
+				color: lifestyle[200],
+			};
+		case Pillar.Opinion:
+			return {
+				background: opinion[200],
+				color: opinion[200],
+			};
+		default:
+			return {
+				background: news[200],
+				color: news[200],
+			};
+	}
+}
+
+const bylineStyle = (colour) => css`
+	color: ${text.ctaPrimary};
+	font-style: italic;
+	${headline.xxsmall()}
+	line-height: 115%;
+
+	a {
+		font-weight: 700;
+		${from.desktop} {
+			color: ${colour};
+		}
+		${until.desktop} {
+			color: ${text.ctaPrimary};
+		}
+		text-decoration: none;
+		font-style: normal;
+		:hover {
+			text-decoration: underline;
+			background: #ffe500;
+			color: #121212;
+		}
+	}
+`;
+
+const dateStyle = (colour) => css`
+	color: ${colour};
+	${textSans.xsmall()}
+	:hover {
+		cursor: pointer;
+	}
+
+	${from.desktop} {
+		color: ${palette.neutral[20]};
+	}
+`;
+
+// I would normally use the Lines component
+const lines = css`
+	background-image: repeating-linear-gradient(
+		to bottom,
+		#dcdcdc,
+		#dcdcdc 0.0625rem,
+		transparent 0.0625rem,
+		transparent 0.25rem
+	);
+	background-repeat: repeat-x;
+	background-position: top;
+	background-size: 0.0625rem 0.8125rem;
+	padding-top: 0.9375rem;
+	margin-bottom: 0.375rem;
+`;
+
+const textContainer = css`
+	margin-bottom: ${space[3]}px;
+	${from.desktop} {
+		margin-bottom: ${space[6]}px;
+	}
+`;
+
+const detailStyle = css`
+	details > summary {
+		list-style: none;
+	}
+	details > summary::-webkit-details-marker {
+		display: none;
+	}
+`;
+
+const containerStyle = (background) => css`
+	${background};
+	width: 100%;
+	padding: ${space[3]}px ${space[5]}px ${space[5]}px ${space[3]}px;
+	${until.phablet} {
+		padding-left: ${space[3]}px;
+		padding-right: ${space[3]}px;
+	}
+
+	${from.desktop} {
+		background-color: ${palette.neutral[100]};
+		width: 13.75rem;
+	}
+`;
+
+export const MetaData = ({
+	theme,
+	palette,
+	byline,
+	tags,
+	authors,
+	primaryDateline,
+	secondaryDateline,
+}: Props) => {
+	// Remember how to get array of contributors
```

Comment:
Again, maybe think about how this comment could explain better what's happening here - is this a `TODO` and if so why are we not implementing the line below currently?